sources/packages/backend/apps/test-db-seeding/src/db-seeding/institution/basic-seeding/create-institutions-and-authentication-users.ts

```diff
@@ -97,9 +97,12 @@ export class CreateInstitutionsAndAuthenticationUsers {
           user.userRole,
         );
       // Check if a default location need to be created.
-      // Only regular users must be associated with a location.
+      // Only read-only and regular users must be associated with a location.
       let location: InstitutionLocation;
-      if (user.userType === InstitutionUserTypes.user) {
+      if (
+        user.userType === InstitutionUserTypes.user ||
+        user.userType === InstitutionUserTypes.readOnlyUser
+      ) {
```

Comment:
Can we check "if not an admin"? It wold allow new roles to be added in the future without a need to change this code again.